.docsettings.yml

```diff
@@ -22,6 +22,29 @@ required_readme_sections:
   - ^Troubleshooting$
   - ^Next steps$
   - ^Contributing$
+
+known_presence_issues:
+  - ['CHANGELOG.md','#1583']
+  - ['sdk/batch/batch/CHANGELOG.md','#1583']
+  - ['sdk/storage/storage-datalake/CHANGELOG.md','#1583']
+  - ['sdk/storage/storage-datalake/CHANGELOG.md','#1583']
+  - ['sdk/batch/batch/CHANGELOG.md','#1583']
+  - ['sdk/template/template/CHANGELOG.md','#1583']
+  - ['sdk/batch/batch/CHANGELOG.md','#1583']
+  - ['sdk/graphrbac/graph/CHANGELOG.md','#1583']
+  - ['sdk/servicebus/service-bus/samples/CHANGELOG.md','#1583']
+  - ['sdk/applicationinsights/applicationinsights-query/CHANGELOG.md','#1583']
+  - ['sdk/test-utils/recorder/CHANGELOG.md','#1583']
+  - ['sdk/eventhub/testhub/CHANGELOG.md','#1583']
+  - ['sdk/storage/storage-file-datalake/samples/CHANGELOG.md','#1583']
+  - ['sdk/eventgrid/eventgrid/CHANGELOG.md','#1583']
+  - ['sdk/common/smoke-test/CHANGELOG.md','#1583']
+  - ['common/smoke-test/CHANGELOG.md','#1583']
+  - ['sdk/servicefabric/servicefabric/CHANGELOG.md','#1583']
+  - ['sdk/operationalinsights/loganalytics/CHANGELOG.md','#1583']
+  - ['sdk/storage/storage-file-share/samples/typescript/CHANGELOG.md','#1583']
+  - ['sdk/storage/storage-file-share/samples/javascript/CHANGELOG.md','#1583']
```

Comment:
Consider starting a Changelog for any of these that are track 2 libraries.

Reply:
@chidozieononiwu In the above list `sdk/batch/batch/CHANGELOG.md` is mentioned thrice

Can we tweak the checks to not look at `samples` folder?

Reply:
removed multiple entries, and added common to excluded files